src/Sylius/Behat/Context/Setup/UserContext.php

```diff
@@ -155,6 +156,19 @@ private function prepareUserPasswordResetToken(UserInterface $user): void
         $this->userManager->flush();
     }
 
+    /**
+     * @Given /^(I) waited too long, and the link expired$/
+     */
+    public function iWaitedTooMuchTimeToResetMyPassword(UserInterface $user): void
+    {
+        /** @var \DateTime $passwordRequestedAt */
+        $passwordRequestedAt = $user->getPasswordRequestedAt();
+        $passwordRequestedAt->sub(new \DateInterval($this->passwordResetTokenTtl));
```

Comment:
I'd be for not using the exact ttl and instead, either subtracting like a year or duplicating this line so it's still dynamic, date operations tend to be wobbly and might result in random fails due to skip years, daylight saving time date changes, etc

Reply:
Since the TTL default option is not part of any Interface constant, I choose to duplicate the line and add a comment quoting your explanation.